ferrocene/doc/release-notes/src/25.02.0.rst

```diff
@@ -0,0 +1,42 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Ferrocene 25.02.0
+=================
+
+Ferrocene 25.02.0 is the fith major release of Ferrocene.
+
+The highlights of this release are the inclusion of Rust 1.83.0.
+
+New features
+------------
+
+There have been no new features since last release.
```

Comment:
The Rust version bump should be a new feature.

Reply:
Corrected